dev/io.openliberty.webcontainer.servlet.6.1.internal_fat/bnd.bnd

```diff
@@ -0,0 +1,23 @@
+#*******************************************************************************
+# Copyright (c) 2023 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+src: \
+    fat/src
+
+fat.project: true
+
+javac.source: 17
+javac.target: 17
```

Comment:
Should source and target are at 21 as well?

Reply:
@pmd1nh  it will eventually once we can build open liberty with Java 21. Currently we can execute the FATs with 21 but not build.